### PR TITLE
Fully implement stakeholder features 

### LINKED ISSUE #1: Add getDelegatedCCS function
Add a function that receives a stakeholder address as an argument and returns its delegation info.

`(PlatformAddress) => { delegatee: PlatformAddress, quantity: U64 }[]`

### PR DESCRIPTION
**Pull request by foriequal0**
* Create Transaction
   * Transfer
   * Delegate
   * RequestRevoke (Corgi currently doesn't support)
 * Get
   * Balance
   * Stakeholders
   * Delegations
   * Revocations

Resolves: https://github.com/CodeChain-io/codechain-stakeholder-sdk-js/issues/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "codechain-stakeholder-sdk",
```
Comment (joojis) [inline package.json]: You shouldn't change this field because `-js` postfix is redundant when it comes to the NPM package.

File: src/index.ts [hunk 2]
```diff
@@ -65,4 +140,34 @@ export const createTransferCCSTransaction = (
             U64.ensure(quantity).toEncodeObject()
         ])
     });
-};
+}
+
+export function createDelegateCCSTransaction(
+    sdk: SDK,
+    delegator: PlatformAddressValue,
```
Comment (joojis) [inline src/index.ts]: Doesn't it have to be a `delegatee`?

File: src/index.ts [hunk 3]
```diff
@@ -65,4 +140,34 @@ export const createTransferCCSTransaction = (
...
+            DELEGATE_CCS_ACTION_ID,
+            PlatformAddress.ensure(delegator).accountId.toEncodeObject(),
+            U64.ensure(quantity).toEncodeObject()
+        ])
+    });
+}
+
+export function createRequestRevokeTransaction(
+    sdk: SDK,
+    delegator: PlatformAddressValue,
```
Comment (joojis) [inline src/index.ts]: The same here.

File: README.md [hunk 4]
```diff
@@ -68,3 +102,35 @@ sdk.rpc.chain.sendSignedTransaction(signedTx)
...
+const tx = createDelegateCCSTransaction(sdk, "tccq94guhkrfndnehnca06dlkxcfuq0gdlamvw9ga4f", 100);
+const signedTx = tx.sign({ secret: "...", seq: "...", fee: "..." });
+sdk.rpc.chain.sendSignedTransaction(signedTx)
+  .then(txhash => {
+    // txhash: H256
+    ...
+  });
+```
+
+### Request to revoke a delegated stake tokens
```
Comment (joojis) [inline README.md]: Is the article `a` necessary? I'm not sure about it

**Review by joojis — APPROVED** [review]
LGTM

File: README.md (hunk 4 above)
Comment (foriequal0) [reply README.md]: I'll fix them and merge this PR.

**Pull request merged**